php/libraries/Candidate.class.inc

```diff
@@ -613,22 +613,19 @@ class Candidate
     /**
      * Returns first visit for a candidate based on Date_visit
      *
-     * @return string The visit label for the candidate's first visit
+     * @return string|null The visit label for the candidate's first visit
      */
     public function getFirstVisit()
     {
         $factory = NDB_Factory::singleton();
         $db      = $factory->database();
 
         $candID = $this->getCandID();
-
-        $query  = "SELECT Visit_label
-            FROM session s JOIN candidate c ON (c.CandID = s.CandID)
-            WHERE c.CandID=:cid AND Date_Visit IS NOT NULL
-            ORDER BY Date_Visit";
+        $query  = "SELECT Visit_label FROM session 
+          WHERE CandID = :cid AND Date_visit IS NOT NULL 
+          ORDER BY Date_visit, ID ASC LIMIT 1";
         $where  = array('cid' => $candID);
-        $vLabel = $db->pselectOne($query, $where);
-
+        $vLabel = $db->pselectCol($query, $where)[0] ?? null;
```

Comment:
pselectCol will return an Associative array, so [0] will be always null.
pselectOne will return the value of the first column of the first row.
https://github.com/kongtiaowang/Loris/pull/168/files